module-apps/application-settings/windows/advanced/ColorTestWindow.cpp

```diff
@@ -76,7 +76,7 @@ namespace gui
         if (scheme != currentColorScheme) {
             currentColorScheme = scheme;
             application->bus.sendUnicastSync(
-                std::make_shared<service::gui::ChangeColorScheme>(std::move(scheme)), service::name::gui, 100);
+                std::make_shared<service::gui::ChangeColorScheme>(scheme), service::name::gui, 100);
```

Comment:
why the scheme is not moved here? It has already been copied to `currentColorScheme` so it shouldn't be an issue

Reply:
scheme is a const variable in this context, so it will be copied despite using std::move, as it cannot be moved

Reply:
true :D